pkg/dtest_k3s/k3s.go

```diff
@@ -298,13 +310,15 @@ func Kubeconfig(ctx context.Context) string {
 
 	K3sUp(ctx)
 
-	for {
+	dlog.Printf(ctx, "Polling for k3s to be ready...")
+	for ctx.Err() == nil {
```

Comment:
This isn't quiet error handling, but it means that this doesn't block forever if the ctx dies.